src/main/java/com/opinta/mapper/ClientMapper.java

```diff
@@ -5,8 +5,9 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.mapstruct.Mappings;
+import org.springframework.stereotype.Component;
 
-
+@Component
```

Comment:
There was annoying error from ide, that said that it can't inject bean. Adding these @Component annotations fixed the problem.